__tests__/auth.spec.js

```diff
@@ -67,4 +69,107 @@ describe('AUTH API ENDPOINTS', () => {
+            const { status, body } = await server()
+                .post('/api/v1/auth/reset')
+                .send({ email: 'ba@ gmail' });
+
+            expect(status).toBe(422);
+            expect(body).toMatchSnapshot();
+        });
+
+        it('should not send mail if user is not found', async () => {
+            const { status, body } = await server()
+                .post('/api/v1/auth/reset')
+                .send({ email: 'bambam@gmail.com' });
+
+            expect(status).toBe(404);
+            expect(body).toMatchSnapshot();
+        });
+
+        it('should send email if user is valid', async () => {
+            const user = getUser();
+            await createUser(user);
+            emailReset = user.email;
+
+            const { status, body } = await server()
+                .post('/api/v1/auth/reset')
+                .send({ email: emailReset });
+
+            expect(status).toBe(200);
+            expect(body.data.message).toBe(
+                'Check your email for password reset link.'
+            );
+            expect(Object.keys(body.data)).toMatchSnapshot();
+        });
+    });
+
+    describe('GET PASSWORD RESET api/v1/auth/reset', () => {
+        it('should access reset page with invalid token', async () => {
+            const { status, body } = await server().get(
+                '/api/v1/auth/reset/sdff'
+            );
+
+            expect(status).toBe(422);
+            expect(body.message[0].message).toBe('reset token is invalid');
+            expect(body).toMatchSnapshot();
+        });
+
+        it('should access reset page if token is valid', async () => {
+            const user = await findUser(emailReset);
+            resettoken = user[0].resettoken;
+            const { status } = await server().get(
+                `/api/v1/auth/reset/${resettoken}`
+            );
+
+            expect(status).toBe(200);
+        }, 30000);
+    });
+
+    describe('PATCH PASSWORD RESET api/v1/auth/reset', () => {
+        it('should not reset password if token is invalid', async () => {
+            const { status, body } = await server()
+                .patch('/api/v1/auth/reset')
+                .send({
+                    password: 'qwert12345',
+                    password_confirmation: 'qwert12345',
+                    token: 'resettoken'
+                });
+
+            expect(status).toBe(422);
+            expect(body.message[0].message).toBe('reset token is invalid');
+            expect(body).toMatchSnapshot();
+        });
+
+        it('should not reset password if password is invalid', async () => {
+            const { status, body } = await server()
+                .patch('/api/v1/auth/reset')
+                .send({
+                    password: 'qwert',
+                    password_confirmation: 'qwert12345',
+                    token: resettoken
+                });
+
+            expect(status).toBe(422);
+            expect(body.message[0].message).toBe(
```

Comment:
You are already taking a snapshot of this. If you really want to see the response in the test, you can use an inline snapshot.

Reply:
fixed